nonblock-rpc/rust/core/src/api.rs

```diff
@@ -0,0 +1,13 @@
+use crate::ROUTER;
+
+#[uniffi::export]
+pub fn call(path: String, data: Vec<String>) -> String {
+    let routes = &ROUTER.routes;
+
+    match routes.get(&path) {
+        Some(handler) => {
+            handler.call(data)
+        },
+        None => "Unknown function".to_string(),
```

Comment:
We're currently suppressing errors here. This method should return `Result<String>`, where the error type is a `uniffi` error type instead.